prospector/processing/test_natural_language_processing.py

```diff
@@ -0,0 +1,92 @@
+from processing.natural_language_processing import (
+    extract_cve_references,
+    extract_jira_references,
+    extract_non_nl_terms,
+    extract_path_tokens,
+)
+
+
+def test_extract_non_nl_terms():
```

Comment:
Shall we call them `special terms` or `keywords` or something like that, as we discussed in our last dev. call?